fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/utils/NodePropertiesTools.java

```diff
@@ -203,9 +203,18 @@ public void removeNodeProperty(final Node node, final String propertyName, final
             if (property.isMultiple()) {
                 final AtomicBoolean remove = new AtomicBoolean();
                 final Value[] newValues = stream(node.getProperty(propertyName).getValues()).filter(v -> {
-                    if (v.equals(valueToRemove)) {
-                        remove.set(true);
-                        return false;
+                    String strVal = "";
+                    try {
+                        strVal = v.getString();
+
+                        if (strVal.equals(valueToRemove.getString())) {
+                            remove.set(true);
+                            return false;
+                        }
+
+                    } catch (RepositoryException e) {
```

Comment:
Thanks! I'll put in the UncheckedPredicate.  I don't know if it should continue processing or not. At the very least it would be nice to warn the user that their remove failed, but, being a newbie, I'm not sure what the best mechanism to make that happen would be.